src/states/bootstrapping_peer.rs

```diff
@@ -137,7 +137,7 @@ impl BootstrappingPeer {
         };
 
         self.send_direct_message(&dst, DirectMessage::BootstrapRequest(destination));
-        self.peer_map_mut().connect(dst);
+        self.peer_map_mut().connect(dst.peer_addr, dst);
```

Comment:
might as well only have a single param and do that within the `connect` function?

Reply:
Yep you're right, that would be simpler (and more correct)

Reply:
We don't have the 1-to-many mapping anymore as we did when the key was `PublicId`